source/core/index.ts

```diff
@@ -845,12 +862,28 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 		}
 
 		// HTTPS options
+		if ('rejectUnauthorized' in options) {
+			deprecationWarning('options.rejectUnauthorized', '"options.rejectUnauthorized" is now deprecated, please use "options.https.rejectUnauthorized"');
```

Comment:
Can't we just pass the message here? I don't think the first argument is needed at all :)